src/Models/Question.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Engelsystem\Models;
+
+use Engelsystem\Models\User\User;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+/**
+ * @property integer $id
+ * @property integer $enquirer_id
+ * @property string $question
```

Comment:
Or maybe text? That would be my personal preference and is imho better readable (as its shorter) than `$question->question`

Reply:
✓